app/views/staff/planned_disbursements/_form.html.haml

```diff
@@ -1,4 +1,4 @@
-= f.govuk_error_summary
+= f.govuk_error_summary link_base_errors_to: :financial_quarter
```

Comment:
I've read the [documentation](https://govuk-form-builder.netlify.app/introduction/error-handling/#linking-errors-on-the-base-object-to-specific-fields) on the govuk form builder regarding linking the error to the base. I have a question about this. If there are any other errors on this form (apart from an attempt to create a second `:original` planned disbursement, all the errors will be linked to `:financial_quarter` because that's the recommendation from govuk, right? (They suggest to link them to the first field)

Reply:
Re: my comment above, I'm wondering if this is an error that should ever be shown to the user. My expectation here was the user states what the value for the planned disbursement for a certain activity/quarter should be, and we either create or update `PlannedDisbursement` records to reflect that. Does the current workflow assume this always creates new records, and this is a stepping stone to the design in https://gist.github.com/jcoglan/dcb4e87652fd172a83f9263973cad2da? What is a user expected to do if they see this error on submitting the form?

Reply:
@MariluzHerrera As far as I understand it the errors on `base` will all be linked to `financial_quarter` (if it actually worked!) errors that relate to a specific field would still be linked to the relevent fields as they would be associated in the `errors` object.

@jcoglan I found very easy to see this errors the way the UI works right now, we have am 'add forecast' button that lets the user add a new original forecast, if they were to select a quarter and year of an existing original planned disbursement and submit the form, this would be the error state.

Yes this is the 3rd step towrds a complete forecast workflow, sorry I should have been clearer in the PR message.

I agree it is a dead end for the user right now, but I do feel the priority is to get forecasts working so we can collect them right now.